pkg/operator/sync_kubecontrollermanager_v311_00.go

```diff
@@ -2,25 +2,26 @@ package operator
 
 import (
 	"fmt"
+	"os"
 
-	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/apis/kubecontrollermanager/v1alpha1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
 	appsclientv1 "k8s.io/client-go/kubernetes/typed/apps/v1"
```

Comment:
Just fyi, the convention for these is `<group><version>client`, iow. `appsv1client`.